README.md

```diff
@@ -221,6 +223,119 @@ In the example above, there are three strains: `default` serves content from `ww
 
 Finally, on `proxy.primordialsoup.life`, all content of the old site is being served. This allows you to easily switch back to an old configuration.
 
+## Development - Serving local content
+
+Getting the Helix Development Server to use a local content repository can be done in 2 ways:
+
+### Specify a local content url
+
+This is the out-of-the box setup:
+
+```yaml
+definitions:
+  defaults:
+    - &localRepo "http://localhost/local/default.git"
+
+strains:
+  - name: default
+    url: http://localhost:3000/
+    code: *localRepo
+    content: *localRepo
+    static: *localRepo
+```
+
+The Helix Development server will automatically start a git server that can serve the content from
+the local repository.
+
+### Use the GitHub emulator
+
+When starting the `hlx up` with `--local-repo` argument(s), it instructs the Helix Development Server to
+start a git server that emulates GitHub repositories for a local git repository. All the strains that
+have a _content_ or _static_ url that matches the `origin` of emulated repository are internally reconfigured
+to use the local git server instead.
+
+For the simple case, where only one repository is used for code, content and static just do:
+
+```
+$ hlx up --local-repo
+``` 
+
+#### Multi Strain Example
+
+In the following config, we define 2 repositories:
+
+- `defaultRepo` contains the project's code and the main content
+- `apiRepo` contains additional content; for example the API documentation.
+
+We also define 2 strains, one for each purpose.
+
+```yaml
+definitions:
+  repos:
+    - &defaultRepo https://github.com/helix/welcome.git#master
+    - &apiRepo https://github.com/helix/welcome-api.git#master
+
+strains:
+  - name: api
+    url: https://www.project-helix.io/api
+    code: *defaultRepo
+    content: *apiRepo
+    static: *apiRepo
+
+  - name: default
+    url: https://www.project-helix.io/
+    code: *defaultRepo
+    content: *defaultRepo
+    static: *defaultRepo
+```
+
+Usually, when invoking `hlx up` without any arguments, the Helix Development Server will serve the
+content directly from GitHub. This is not suitable for local development. Also, the `api` strain
+will never be selected, because the `localhost:3000` host header will not match the specified `url` 
+condition.
+
+Starting the server with:  
+
+```
+$ hlx up --host=www.project-helix.io
+```
+
+Solves the latter problem. the `--host` argument internally overrides the `request.header`, so that
```

Comment:
I feel like the `--host` flag should be described in its own dedicated and separate section.

Reply:
I just want to document the bare minimum. I think all this should go somewhere else, and not in the README of this project. but I agree.

Reply:
OK, makes sense.